### PR TITLE
Fixes: reload page after refund, refund failed message with error cod…

### PR DESCRIPTION
**Pull request by nishitlangaliya**

### CODE REVIEW

File: assets/js/admin/meta-boxes-order.js [hunk 1]
```diff
@@ -697,6 +697,7 @@ jQuery( function ( $ ) {
 					$.post( woocommerce_admin_meta_boxes.ajax_url, data, function( response ) {
 						if ( true === response.success ) {
 							wc_meta_boxes_order_items.reload_items();
+							window.location.href = window.location.href;
```
Comment (claudiosmweb) [inline assets/js/admin/meta-boxes-order.js]: This will redirect the user before test if `fully_refunded`...

Comment (nishitlangaliya) [inline assets/js/admin/meta-boxes-order.js]: @claudiosmweb, Yes I agree with your concern. should I tried with different logic here and push code for review and do PR ?

**Comment by nishitlangaliya** [comment]
@claudiosmweb, Yes I agree with your concern. should I tried with different logic here and push code for review and do PR ?

**Comment by nishitlangaliya** [comment]
@claudiosmweb, I have one query in 'assets\js\admin\meta-boxes-order.js' on line no 699. in which condition 'response.data.status' will return 'fully_refunded' ? As I see in debug 'response.data.status' always returns undefined so should we keep this <pre>if ( 'fully_refunded' === response.data.status ) { </pre> ?

**Pull request closed without merging**